action.yml

```diff
@@ -32,6 +32,9 @@ inputs:
   ruby-tracer-version:
     description: 'The version of datadog-ci Ruby gem to use (optional). Defaults to the latest release.'
     required: false
+  go-tracer-version:
+    description: 'The version of Orchestrion to use (optional). Defaults to the latest release.'
```

Comment:
Orchestrion is a DataDog thing, isn't it? I wonder if we should add some brief explanation, or a link to the docs. It seems possible that a random Go developer has no idea what Orchestrion is